.github/workflows/ci-eclint.yml

```diff
@@ -0,0 +1,32 @@
+name: ci-eclint
```

Comment:
It would be great if we can add it to https://github.com/apache/apisix/blob/042ce5c1a6a4e040d6d6924f2d86e5534b730134/.github/workflows/lint.yml#L5, so that we can reduce the number of workflows we need to maintain.